pkg/agent/flannel/setup.go

```diff
@@ -125,11 +126,12 @@ func waitForPodCIDR(ctx context.Context, nodeName string, nodes typedcorev1.Node
 			break
 		}
 	}
-	logrus.Info("PodCIDR assigned for node " + nodeName)
+	logrus.Info("flannel assigned PodCIDR for node " + nodeName)
```

Comment:
nit - flannel found the assignment from the node IPAM, it did not assign it itself, I believe?
```suggestion
	logrus.Info("Flannel found PodCIDR assigned for node " + nodeName)
```